src/Rule/EnumMulti.php

```diff
@@ -53,7 +53,7 @@ public function setParameter($params)
 	public function getMessageParameters()
 	{
 		return array(
-			'values' => implode('|', $this->getParameter()),
+			'value' => implode('|', $this->getParameter()),
```

Comment:
It's potentially going to break any existing custom messages that people have set for the rule. I'd advise against changing the name.